src/StatusQ/Components/StatusChatList.qml

```diff
@@ -96,31 +92,19 @@ Column {
                 StatusChatListItem {
 
                     id: statusChatListItem
-
-                    property string profileImage: ""
-
                     opacity: dragSensor.active ? 0.0 : 1.0
-                    Component.onCompleted: {
-                        if (typeof statusChatList.profileImageFn === "function") {
-                            profileImage = statusChatList.profileImageFn(model.chatId || model.id) || ""
-                        }
-                    }
                     originalOrder: model.position
-                    chatId: model.chatId || model.id
-                    categoryId: model.categoryId || ""
-                    name: !!statusChatList.chatNameFn ? statusChatList.chatNameFn(model) : model.name
-                    type: model.chatType
-                    muted: !!model.muted
-                    hasUnreadMessages: !!model.hasUnreadMessages || model.unviewedMessagesCount > 0
-                    hasMention: model.mentionsCount > 0
-                    badge.value: model.chatType === StatusChatListItem.Type.OneToOneChat ?
-                        model.unviewedMessagesCount || 0 :
-                        model.mentionsCount || 0
-                    selected: (model.chatId || model.id) === statusChatList.selectedChatId
-
-                    icon.color: model.color || ""
-                    image.isIdenticon: !!!profileImage && !!!model.identityImage && !!model.identicon
-                    image.source: profileImage || model.identityImage || model.identicon || ""
+                    chatId: model.itemId
+                    categoryId: model.itemId
+                    name: model.name
+                    type: !!model.type ? model.type : StatusChatListItem.Type.CommunityChat
+                    muted: model.muted
+                    hasNotifications: model.hasNotification
```

Comment:
Actually one more question here: shouldn't this be `hasNotifiactions` ?
And also... isn't `hasUnreadMessages` more correct? Not wanna bikeshed here, but... having notifications is one thing (that's why we have the `notificationCount`, having unread messages another)

Reply:
> Actually one more question here: shouldn't this be hasNotifiactions ?

@PascalPrecht on the BE side it's exposed as `hasNotification`, not a problem at all to update it.

> isn't hasUnreadMessages more correct?

@PascalPrecht depends on how you're looking into it :) `hasNotifications` is somehow general I would say, so if we're talking about unread messages, then `hasNotifications` signifies a notification about unread messages, if we're talking about mentions then it signifies that there are mention notifications, if we're talking about replies then it signifies that there are replies notifications.

> having notifications is one thing (that's why we have the notificationCount, having unread messages another)

A message which has mention or is a reply to any of my previous messages is in both case unread message. :)

All in all, they are all in real just unread messages but some contains only text, some mentions, some are replies, but all them are still unviewed/unseen/unread messages. So I agree with whatever you say regarding this. Just say how would you like to see this and I will update it?

Reply:
I personally prefer `hasUnreadMessages`. We already know if we have notifications (`noticiationsCount > 0`). That should make it very explicit.